code/auth-service/src/dataaccess/userRolesRepository.spec.js

```diff
@@ -0,0 +1,52 @@
+import getRoles from './userRolesRepository';
+import database from '../config/database';
+import databaseMock from './testUtil/databaseMock';
+
+const wrapDocument = document => ({
+  ...document,
+  toObject: () => document,
+});
+
+const testUserRoles = [
+  wrapDocument({
```

Comment:
I know I wrote this.... but. It might be cleaner to leave `testUserRoles` as the unwrapped array and then wrap all documents by
`const mappedDocuments = testUserRoles.map(wrapDocument);`
..at least I think that will work.